sdk/eventgrid/eventgrid/package.json

```diff
@@ -99,7 +99,7 @@
   "devDependencies": {
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
-    "@azure/service-bus": "^7.0.0",
+    "@azure/service-bus": "^7.3.0",
```

Comment:
why is this needed?

Reply:
And if this change is needed, it should probably go into a separate PR.  I can't think of any reason the perf tests should depend on this change.

Reply:
So that all the packages are using the same version of service-bus?

Reply:
We shouldn't update a runtime dep for this reason.  I would just have the perf test depend on `^7.0.0` as well.   It will use the source version by default either way.

Reply:
Updated 🙂